.github/workflows/build_and_test.yml

```diff
@@ -1162,3 +1162,9 @@ jobs:
           cd ui-test
           npm install --save-dev
           node --experimental-vm-modules node_modules/.bin/jest
+
+  maven-test:
+    name: "Run Maven Test"
+    permissions:
+      packages: write
+    uses: ./.github/workflows/maven_test.yml
```

Comment:
Could you revert this leftover, @panbingkun ?

Reply:
Done.